src/iad.cs

```diff
@@ -199,7 +200,8 @@ partial interface IAdPreroll {
 #endif
 
 	[Deprecated (PlatformName.iOS, 10, 0)]
-	[Category, BaseType (typeof (UIViewController))]
+	[Category (ignoreStaticWarnings: true)]
```

Comment:
That's a case where we should not warn, i.e. the `[Static]` is `[Internal]` so our public API is fine.